oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Configuration.java

```diff
@@ -260,6 +260,27 @@
                     "are separated with '::'. Example: -Doak.documentstore.persistentCacheIncludes=/content::/var")
     String[] persistentCacheIncludes() default {"/"};
 
+    @AttributeDefinition(
+            name = "Full GC Include Paths",
+            description = "Paths which should be included in full garbage collection." +
+                    "Empty value means all paths are included. " +
+                    "Any path which is added to both include & exclude paths, " +
+                    "would be removed from included paths." +
+                    "Note that this value can be overridden with a system property " +
+                    "'oak.documentstore.fullGCIncludes' where paths " +
+                    "are separated with '::'. Example: -Doak.documentstore.fullGCIncludes=/content::/var")
+    String[] fullGCIncludes() default {};
+
+    @AttributeDefinition(
+            name = "Full GC Exclude Paths",
+            description = "Paths which should be excluded from full Garbage collection." +
+                    "Empty value means no paths are excluded." +
+                    "Any path added to excluded list would be removed from include paths (if present)." +
+                    "Note that this value can be overridden with a system property " +
+                    "'oak.documentstore.fullGCExcludes' where paths " +
+                    "are separated with '::'. Example: -Doak.documentstore.fullGCExcludes=/content::/var")
+    String[] fullGCExcludes() default {};
```

Comment:
was wondering if the config should be called `fullGCExcludePaths` - a bit more explicit, wdyt?

Reply:
fixed in https://github.com/apache/jackrabbit-oak/pull/1551/commits/55f310e2d65cfe3208ff7688ce93787eccb0cfba